src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -1,65 +1,201 @@
+                <td> @GetItemType(assetItem)</td>
+                <td>@assetItem.Room.Name</td>
+                <td>@assetItem.Note</td>
+                <td>@assetItem.ItemNumber</td>
+                <td>@assetItem.Shop</td>
+                <td>@GetCas(assetItem)</td>
+                <td>
+                    @if (GetHazards(assetItem).Count == 0)
+                    {
+                        <p>-</p>
+                    }
+                    @foreach (var hazard in GetHazards(assetItem))
+                    {
+                        @hazard.Name
+                        @hazard.ImagePath
+                    }
+                </td>
+                <td>@GetUnit(assetItem)</td>
+                <td>@GetManufacturer(assetItem)</td>
+                <td>@GetSerialNumber(assetItem)</td>
+                <td>
+                    <button type="button"
+                            class="btn">
+                        <i class="bi bi-pencil-square"></i>
+                    </button> <!-- TODO: button needs to be added properly -->
+                </td>
+            </tr>
         }
-    </div>
-</div>
+        </tbody>
+    </table>
 
-<!-- pagination -->
-<nav aria-label="Page navigation example">
-    <ul class="pagination">
-        <li class="page-item disabled">
-            <button class="page-link">
-                🡸
-            </button>
-        </li>
-        <li class="page-item active"
-            aria-current="page">
-            <button class="page-link">
-                1
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                2
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                3
-            </button>
-        </li>
-        <li class="page-item">
-            <button class="page-link">
-                🡺
-            </button>
-        </li>
-    </ul>
-</nav>
+    <!-- pagination -->
+    <nav aria-label="Page navigation example">
+        <ul class="pagination d-flex justify-content-center">
+            <li class="page-item disabled">
+                <button class="page-link">
+                    🡸
+                </button>
+            </li>
+            <li class="page-item active"
+                aria-current="page">
+                <button class="page-link">
+                    1
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    2
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    3
+                </button>
+            </li>
+            <li class="page-item">
+                <button class="page-link">
+                    🡺
+                </button>
+            </li>
+        </ul>
+    </nav>
+</div>
 
 @code {
```

Comment:
🌱♻️ The code in this block seems substantial; we could consider extracting it into a companion class

Reply:
Which code block exactly should be extracted?

Reply:
The complete `@code` block into a file like the custom CSS.
It's called `AssetList.razor.cs` I think and done like mentioned [here](https://learn.microsoft.com/de-de/aspnet/core/blazor/components/?view=aspnetcore-9.0#partial-class-support).